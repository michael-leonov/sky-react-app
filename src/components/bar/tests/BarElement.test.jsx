import { render } from '@testing-library/react'
import BarElement from '../BarElement'

describe('<BarElement />', () => {
  describe('Snapshots tests', () => {
    it('should render "play bar element" successfully and match snapshot', () => {
      const typeBarElem = 'play'
      const { container } = render(
        <BarElement>
          <svg>
            <use xlinkHref={`sprite.svg#icon-${typeBarElem}`} />
          </svg>
        </BarElement>
      )

      expect(container).toMatchSnapshot()
    })
  })
})
