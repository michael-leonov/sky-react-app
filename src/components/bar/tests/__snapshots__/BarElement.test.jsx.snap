// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BarElement /> Snapshots tests should render "play bar element" successfully and match snapshot 1`] = `
<div>
  <div
    class="sc-csuSiG cTjzTl"
  >
    <svg
      class="sc-eDWCr hevNYN"
    >
      <use
        xlink:href="sprite.svg#icon-undefined"
      />
    </svg>
  </div>
</div>
`;
